### PR TITLE
[CFDP-1191] Filters when Zoom Buttons show up

### PR DESCRIPTION
**Pull request by asleepingpanda**
* video calls require a user to be logged in to access
* video calls only show when the group is currently active
* duration of the Group is defaulted to 4 hours from the start time

### 1. What does this PR do, or why is it needed?
Group Video buttons should only be visible when the Group is currently meeting

### 2. What design trade-offs/decisions were made?
Groups in Rock don't have a duration, so we've defaulted the duration to 4 hours from the start time of the group

### 3. How do I test this PR?
Query for a bunch of Groups to see if the video calls show up as expected

### 4. What automated tests did you write?
n/a

### CODE REVIEW

File: src/data/checkinable/data-source.js [hunk 1]
```diff
@@ -69,6 +69,8 @@ export default class Checkinable extends RockApolloDataSource {
     const { Schedule } = this.context.dataSources;
     const { nextStart, startOffset, endOffset } = await Schedule.parseById(scheduleId);
 
+    console.log({ nextStart, startOffset, endOffset });
```
Comment (dzwood) [inline src/data/checkinable/data-source.js:72]: Remove console log?

File: src/data/checkinable/data-source.js [hunk 2]
```diff
@@ -100,6 +102,9 @@ export default class Checkinable extends RockApolloDataSource {
        *  for check in. This would mean that this schedule is valid between 6:30am
        *  and 7:30am
        */
+
+      console.log({ isActive, returnData });
```
Comment (dzwood) [inline src/data/checkinable/data-source.js:106]: Remove console log?

File: src/data/schedule/data-source.js [hunk 3]
```diff
@@ -398,6 +399,9 @@ export default class Schedule extends RockApolloDataSource {
 
   timeIsInSchedules = async ({ ids, time }) => {
     const times = await this.getOccurrencesFromIds(ids);
+
+    console.log({ times });
```
Comment (dzwood) [inline src/data/schedule/data-source.js:403]: Remove console log?

**Review by dzwood — APPROVED** [review]
Everything looks good to me, I noticed a few console logs. Wasn't sure if you had them there purposely or not.

**Comment by asleepingpanda** [comment]
@dzwood all console logs have been removed

**Pull request merged**